porcupine/plugins/python_tools.py

```diff
@@ -63,6 +63,21 @@ def format_code_in_textwidget(tool: str, tab: tabs.FileTab) -> None:
             tab.textwidget.replace("1.0", "end - 1 char", after)
 
 
+black_format_tab = actions.register_filetab_action(
+    name="Black Format Tab",
+    description="Autoformat open tab using Black",
+    callback=partial(format_code_in_textwidget, "black"),
+    availability_callback=lambda tab: isinstance(tab, tabs.FileTab),
```

Comment:
`FileTabAction` is never available for tabs other than `FileTab`s:

```suggestion
    availability_callback=lambda tab: True,
```

Reply:
good call